src/api/blockchain_test.go

```diff
@@ -784,7 +784,7 @@ func TestGetBlocks(t *testing.T) {
 			body: &httpBody{
 				Seqs: "1,2,3",
 			},
-			seqs: []uint64{1, 2, 3},
+			seqs:                  []uint64{1, 2, 3},
```

Comment:
Do you have a different `goformat` tool that's causing this?

Reply:
not sure why this happens, I dont think you can configure this?
I am on `go1.11.2 darwin/amd64`